src/rendering/screenspacerenderable.cpp

```diff
@@ -190,6 +196,9 @@ namespace {
         // [[codegen::verbatim(ScaleInfo.description)]]
         std::optional<float> scale;
 
+        // [[codegen::verbatim(UseRadiusAzimuthElevationInfo.description)]]
```

Comment:
This refers to the `UseRadiusAzimuthElevationInfo` description and not the `GammaInfo` description